elastic/apm-server
# Service specific source maps

## Motivation/summary

support configuring source maps from kibana and fetching them from fleet server.

closes https://github.com/elastic/apm-server/issues/5002

## Checklist




## How to test these changes

Run elasticsearch + and elastic-agent in fleet mode.

Start kibana with the code from https://github.com/elastic/kibana/issues/95393 has been completed, and apm-server with the fleet-server sourcemap code.

- Upload a sourcemap via kibana
- Confirm an updated config with the new `source_maps` block has been pushed to apm-server
- Confirm ingesting rum traces for the uploaded sourcemap get mapped appropriately

## Related issues

depends on https://github.com/elastic/kibana/issues/95393
